js/index.js

```diff
@@ -44,4 +44,66 @@ const designTools = [
       li.textContent = tool;
       toolsList.appendChild(li);
     });
-  });
\ No newline at end of file
+  });
+
+
+
+  
+
+
+const messageForm = document.querySelector('.contact-form');
+if (messageForm) {
+    messageForm.addEventListener('submit', function(event) {
+        event.preventDefault(); 
+
+        
+        const userName = event.target.name.value;
+        const userEmail = event.target.email.value;
+        const userMessage = event.target.message.value;
+
+        const messageSection = document.getElementById('messages');
+        const messageList = messageSection.querySelector('ul');
+        const newMessage = document.createElement('li');
+
+        newMessage.innerHTML = `
+            <a href="mailto:${userEmail}">${userName}</a> 
+            <span>${userMessage}</span>
+            <button type="button" class="edit-button">edit</button>
+            <button type="button" class="remove-button">remove</button>
+        `;
+
+        messageList.appendChild(newMessage);
+
+        if (messageList.children.length > 0) {
```

Comment:
In my opinion, this if condition given it will always be true not needed to make code easier to understand?